src/app/courses/components/new-corse-authors/new-corse-authors.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="input-wrapper">
+  <label for="new-course__input">
```

Comment:
It's a separate component. It should have its own class. Something like 'authors'. You can rename 'input-wrapper' to 'authors' and then it will be 'authors__input', 'authors__label', etc.
'new-course' styles from the parent component are not visible here because of view encapsulation.

Also, 
1. You have a typo in the name of the component
2. The name of the component should be more abstract. Like 'authors-input'. 

When you create a component, you should always consider that the component can be used in some other place. So, the name should not contain any information about what page it is located on, or what parent component it has.